static/js/subs.js

```diff
@@ -727,6 +727,41 @@ $(function () {
         update_announce_stream_state();
     });
 
+    // Search People
+    $(document).on('input', '.add-user-list-filter', function (e) {
+        var users = people.get_rest_of_realm();
+
+        var user_list = $(".add-user-list-filter");
+        if (user_list === 0) {
+            return;
+        }
+        var search_term = user_list.expectOne().val().trim();
+        var search_terms = search_term.toLowerCase().split(",");
+        search_terms = _.map(search_terms, function (s) {
+            return s.trim();
+        });
+
+        _.each(users, function (user) {
+            var names = user.full_name.toLowerCase().split(/\s+/);
+            names = _.map(names, function (s) {
+                return s.trim();
+            });
+
+            _.any(search_terms, function (search_term) {
+                $("label[for='" + user.email + "']").css({"display":"none"});
```

Comment:
This does not work as designed. Because you're resetting the CSS to `display: none` for each search term, only the last search term in a comma-delimited search query has any effect